index/secondary-index/src/test/scala/org/apache/carbondata/spark/testsuite/secondaryindex/TestCreateIndexTable.scala

```diff
@@ -370,6 +370,38 @@ class TestCreateIndexTable extends QueryTest with BeforeAndAfterAll {
     }
   }
 
+  test("index creation on long string columns") {
+    sql("drop table if exists si_table")
```

Comment:
just create a small table which has varchar column, no need to do any dataload here, as intension is just to throw an exception for long string columns.

Reply:
done